README.md

```diff
@@ -1 +1,149 @@
-# Microverse-capstone-one
\ No newline at end of file
+# Microverse-capstone-one<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [Microverse-capstone-one](#microverse-capstone-one)
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 First Capstone Project ](#-first-capstone-project-)
+  - [Project Description\*\*](#project-description)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Install](#install)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 First Capstone Project <a name="about-project"></a>
+
+## Project Description**
+**The Microverse first Capstone Project aims to test understanding of micronaunts, to know how much students understands all concepts learned so far
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> This project was built using the following technologies:
+
+<details>
+  <summary>HTML</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/html/" target="_blank">html</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>CSS</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/css/" target="_blank">css</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Javascript</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/javascript/" target="_blank">Javascript</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Mobile responsiveness**
+- **Information about my work experience and educational background**
+- **Detailed project descriptions with links to code repositories and live demos**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> In order to start with this project:
+
+To get get a local copy up and running, you can follow these simple steps:
+
+
+### Install
+Clone the repository to your local machine using the following command:
+
+```sh
+  git clone [github.com/Paulinelali](https://github.com/Paulinelali/Microverse-capstone-one.git)
+```
+Navigate to the project directory:
+
+Open the index.html file in your web browser to view the project:
+
+ open index.html
+
+You can now make changes to the project as desired and push them back to the repository using Git.
+
```

Comment:
- [ ] It is required that your project should be deployed. That has not been done yet. Kindly deploy your project and update your readme file with its link.

### Project Requirement

![project-rqt](https://github.com/Paulinelali/Microverse-capstone-one/assets/26597432/39c1f715-3b25-41ee-b083-d75c707a00b9)

Reply:
@Qoosim Thank you so much for the time to review my project and you gave well-defined instructions for things that need correction😎 I wish you will be available to review once I complete the required changes